MSI-2084 Introduce tests for 'GetNearbySourcesByPostcode'

vadimjustus opened the pull request:
Introduces integration tests for `\Magento\InventoryInStorePickup\Model\DistanceProvider\Offline\GetNearbySourcesByPostcode` class.

### Fixed Issues (if relevant)
1. magento-engcom/msi#2084: Cover nearest inventory source selection with integration tests

### Manual testing scenarios (*)
1. Running integration tests for `InventoryInStorePickup` module

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

vadimjustus committed:
MSI-2084 Introduce tests for 'GetNearbySourcesByPostcode'

IvanPletnyov commented:
I've run this test, test was passed.

maghamed commented:
hey @vadimjustus could you pleasee look at integration tests, looks like they are broken after your changes

vadimjustus committed:
MSI-2084 Fix integration tests on trvis-ci

ishakhsuvarov commented on app/code/Magento/InventoryInStorePickup/Test/_files/inventory_geoname.php:
```diff
@@ -0,0 +1,222 @@
+	('IT', '45030', 'Ceneselli', 'Veneto', 'RO', 45.0138, 11.3694),
+	('IT', '09040', 'San Nicolo\' Gerrei', 'Sardegna', 'CA', 39.2071, 9.1074),
+	('IT', '80022', 'Arzano', 'Campania', 'NA', 40.9096, 14.2652),
+	('IT', '89020', 'Melicucca\'', 'Calabria', 'RC', 38.3031, 15.8815),
```
`'Melicucca\''` might be a typo

ishakhsuvarov committed:
magento-engcom/msi#2091: Merge branch 'store-pickup' of github.com:magento-engcom/msi into msi-2084-introduce-integration-tests-for-store-pickup-ssa

Conflicts:
	app/code/Magento/InventoryInStorePickup/Model/DistanceProvider/Offline/GetNearbySourcesByPostcode.php
	app/code/Magento/InventoryInStorePickup/composer.json
	app/code/Magento/InventoryInStorePickup/etc/module.xml

malesh-v committed:
Introduce tests for 'GetNearbySourcesByPostcode'

ishakhsuvarov committed:
magento-engcom/msi#2091: Merge branch 'store-pickup' of github.com:magento-engcom/msi into msi-2084-introduce-integration-tests-for-store-pickup-ssa

ishakhsuvarov committed:
magento-engcom/msi#2091: Integration tests for GetNearbySourcesByPostcode
 - table name adjusted

ishakhsuvarov commented:
`continuous-integration/travis-ci/push` was launched by mistake and cancelled.

ghost commented:
Hi @vadimjustus, thank you for your contribution! </br>Please, complete [Contribution Survey](https://opensource.magento.com/survey/contribution/magento-engcom+msi+2091), it will take less than a minute. </br>Your feedback will help us to improve contribution process.
